sycl/test/graph/graph-record-dotp.cpp

```diff
@@ -24,9 +24,7 @@ int main() {
   float beta = 2.0f;
   float gamma = 3.0f;
 
-  sycl::property_list properties{
-      sycl::property::queue::in_order(),
-      sycl::ext::oneapi::property::queue::lazy_execution{}};
+  sycl::property_list properties{sycl::property::queue::in_order()};
```

Comment:
Think we should remove the `in_order` property since we're trying to use `depends_on` for edges